src/sas/qtgui/MainWindow/GuiManager.py

```diff
@@ -60,6 +60,9 @@
 
 logger = logging.getLogger(__name__)
 
+# Expose the loaded perspectives for easy linking between perspective methods/inputs and plots
+LOADED_PERSPECTIVES = {}
```

Comment:
Global variables are contraindicated, especially in this case where they lead to circular imports.

I do not know the qt gui code well enough to code it, but it seems to me that you must be able to find the graph associated with the model (otherwise you wouldn't be able to update the theory plot when parameters change in the fitting perspective). Can you use this mechanism to find the plot associated with the perspective that needs a Q range? Then you can call the Q range selector on the specific plot with a callback to adjust low/high values when the interactors are moved. In particular, your Q range slider code won't need to rummage through the perspective tabs to find the associated perspective and it won't need to know anything about the structure of the perspective to set up the associated getter/setter/input. It should also work well with a linear fit tool, which wants to use the data for selecting the range but not the associated perspective.